cyclonedx/model/bom_meta.py

```diff
@@ -0,0 +1,187 @@
+from datetime import datetime, timezone
+from typing import Iterable, Optional
+
+from sortedcontainers import SortedSet
+
+from cyclonedx.model import LicenseChoice, OrganizationalContact, OrganizationalEntity, Property, ThisTool, Tool
+from cyclonedx.model.component import Component
+
+
+class BomMetaData:
```

Comment:
:question: 
this looks like it was refactored to an own file/module. what is the purpose? 
why was it needed to be moved? whats the benefit of yet another module? are there any other changes?

Reply:
Hi this is due to dependency recursion while type check:
cyclonedx/model/bom.py
```python
from ..parser import BaseParser
```
cyclonedx/parser/__init__.py
```python
from ..model.bom_meta import BomMetaData
```